lighthouse-core/lib/dependency-graph/simulator/network-analyzer.js

```diff
@@ -198,8 +198,8 @@ class NetworkAnalyzer {
       if (!Number.isFinite(timing.receiveHeadersEnd) || timing.receiveHeadersEnd < 0) return;
       if (!record.resourceType) return;
 
-      const serverResponseTimePercentage = SERVER_RESPONSE_PERCENTAGE_OF_TTFB[record.resourceType]
-        || DEFAULT_SERVER_RESPONSE_PERCENTAGE;
+      const serverResponseTimePercentage =
+      SERVER_RESPONSE_PERCENTAGE_OF_TTFB[record.resourceType] || DEFAULT_SERVER_RESPONSE_PERCENTAGE;
```

Comment:
```suggestion
      const serverResponseTimePercentage =
        SERVER_RESPONSE_PERCENTAGE_OF_TTFB[record.resourceType] || DEFAULT_SERVER_RESPONSE_PERCENTAGE;
```

Reply:
heh yeah i had touched this one manually but doing that change goes over 100chars :) 

anyway, just added another break after the `||` to sort it, along with those indents.